src/colorize.cr

```diff
@@ -128,15 +128,16 @@ module Colorize
   # ```
   #
   # NOTE: This is by default disabled on non-TTY devices because they likely don't support ANSI escape codes.
-  # This will also be disabled if the environment variable `TERM` is "dumb".
-  class_property? enabled : Bool = true
+  # This will also be disabled if the environment variable `TERM` is "dumb" or `NO_COLOR` contains any value.
+  class_property? enabled : Bool { !ENV.has_key?("NO_COLOR") }
 
   # Makes `Colorize.enabled` `true` if and only if both of `STDOUT.tty?`
   # and `STDERR.tty?` are `true` and the tty is not considered a dumb terminal.
   # This is determined by the environment variable called `TERM`.
   # If `TERM=dumb`, color won't be enabled.
+  # If `NO_COLOR` contains any value color won't be enabled conforming to https://no-color.org
```

Comment:
Not sure we need to spam the website here. If we want to let's at least do so consistently at all mentions

Reply:
Wouldn't putting it everywhere be spam?

The website is the spec.  How else could I reference the intended behavior?

Reply:
I mean it's a simple enough "spec". The sentence leading up to the link is essentially already describing everything you need to know.

Reply:
Personal opinion: I think it's fine to leave the link. Even if the description is enough, just by reading that I wouldn't know there's a standard. And knowing that might make the standard more... standard, I guess.

Reply:
I agree. There should be a link to reference this convention. Even if it's rather trivial.